src/app/intelligence/intelligence.component.ts

```diff
@@ -22,12 +24,21 @@ export class IntelligenceComponent implements OnInit {
           for (let action of this.actions){
             if (action.type === 'answer' && action.mood !== 'sabta') {
                this.answer = action.expression;
+               this.visible = true;
+
+               if (this.answer === "I can't feel it in here but did you try using fan? It will help you.") {
```

Comment:
Instead of checking if the answer is a certain value, isn't it better to check the value of the query? To check if it is empty?